samcli/commands/check/resources/Graph.py

```diff
@@ -19,6 +22,7 @@ def __init__(self):
         self.yellow_warnings: List = []
         self.red_warnings: List = []
         self.red_burst_warnings: List = []
+        self.lambda_function_pricing_info: LambdaFunctionPricing = None
 
     def generate(self, lambda_functions: List) -> None:
```

Comment:
I think this logic should be done in init, unless you want to enable generating the same graph multiple times.

Reply:
A graph will only be generated once. I will have that logic called in `init`